### PR TITLE
Initial configuration files for the nova-operator.

### PR DESCRIPTION
**Pull request by rlandy**

### CODE REVIEW

**Comment by Sandeepyadav93** [comment]
/pj-rehearse ack 

We have marked failed jobs as `optional: true`

File: ci-operator/jobs/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    - ^master-
+    cluster: build01
+    context: ci/prow/unit
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openstack-k8s-operators-nova-operator-master-unit
```
Comment (bogdando) [inline ci-operator/jobs/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master-presubmits.yaml:212]: ditto

File: ci-operator/jobs/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master-presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    optional: true
+    rerun_command: /test unit
```
Comment (bogdando) [inline ci-operator/jobs/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master-presubmits.yaml:214]: the same

File: ci-operator/jobs/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master-presubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-credentials-file=/etc/report/credentials
+        - --target=unit
```
Comment (bogdando) [inline ci-operator/jobs/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master-presubmits.yaml:221]: functional

**Review by bogdando — CHANGES_REQUESTED** [review]

**Pull request merged**